neo/Wallets/SQLite/VerificationContract.cs

```diff
@@ -15,6 +15,11 @@ public void Deserialize(BinaryReader reader)
         {
             reader.ReadSerializable<UInt160>();
             ParameterList = reader.ReadVarBytes().Select(p => (ContractParameterType)p).ToArray();
+            foreach (var parameter in ParameterList)
```

Comment:
This could be done in the `Linq` query, and save one `foreach`

Reply:
@shargon how would I do that? I tried something like this
```C#
ParameterList.Select(p => Enum.IsDefined(typeof(ContractParameterType),p) == false).Any(throw new FormatException()); 
```
but it does not allow me to throw an exception. `foreach` can't be that expensive since we break out of it on the first invalid enum, right?

Reply:
Check my changes, you can `throw` inside the linq query